app/posts/cross-programme/2024-06-19-induction-discovery-recap.md

```diff
@@ -1,50 +1,38 @@
 ---
 title: Induction discovery recap
-description: "Between February and May 2024, a multidisciplinary team did a discovery to explore the problems and possible solutions for users in the induction collection journey. "
+description: "Between February and May 2024, a multidisciplinary team did a discovery to explore the problems and possible solutions for users in the induction data collection journey. "
 date: 2024-06-19
 ---
 
-
-
-![image](/cross-programme/induction-discovery-recap/cpd-map.png)
-
-
 ## Problem 
 
-After a teacher has gained QTS (qualified teaching status), they are required to serve a 2-year period of induction in a school. This induction period can be passed or failed by the schools Appropriate Body who are in place to ensure that schools are training new teachers in accordance with the ECF (early career framework) guidelines.  
-
-Only teachers that have passed the induction period can continue to teach. The data on which teachers have served induction and if they have passed, are in progress or failed is currently stored in the Database for Qualified teachers.
+After a teacher has gained QTS (qualified teaching status), they are required to serve a 2-year period of induction in a school. This induction period can be passed or failed by the school's Appropriate Body, who are in place to ensure that schools are training new teachers in accordance with the ECF (early career framework) guidelines.  
 
-| Hello | World |
-| ----- | ----- |
-| Some  | Data  |
-| Some  | Data  |
+Only teachers that have passed the induction period can continue to teach. The data on which teachers have served induction and if they have passed, are in progress or failed is currently stored in the Database for Qualified Teachers (DQT).
 
-The Data for Qualified Teachers (DQT) and Appropriate Body (AB) portal are due to be decommissioned, which means that induction data will need to be collected & stored differently in future. 
+The DQT and Appropriate Body portal are due to be decommissioned, which means that induction data will need to be collected & stored differently in future. 
```

Comment:
There's also guidance on how to define acronyms in the GOVUK content guide I linked you to.

Basically, always define them, only define them once when they're first introduced, and define them at the end of the sentence. :)